dev/domain-controller/new-admin-vm/Install-CAM.ps1

```diff
@@ -394,23 +387,15 @@ Configuration InstallCAM
 
 				Write-Host "Setting Paths and Tomcat environment"
 
-
-
-				$Reg = "Registry::HKLM\System\CurrentControlSet\Control\Session Manager\Environment"
-
-				$NewPath = (Get-ItemProperty -Path "$Reg" -Name PATH).Path
-
-				#put tomcat path at the end
+				$NewPath = $env:Path
 				if ($NewPath -notlike "*"+$CatalinaBinLocation+"*")
 				{
-				  $NewPath= $NewPath + ";" + $CatalinaBinLocation
+				    #put tomcat path at the end
+				    $NewPath= $NewPath + ";" + $CatalinaBinLocation
 				}
 
-				Set-ItemProperty -Path "$Reg" -Name PATH -Value $NewPath
-				Set-ItemProperty -Path "$Reg" -Name CATALINA_HOME -Value $CatalinaHomeLocation
-
-				# set the current environment CATALINA_HOME as well since the service installer will need that
-				$env:CATALINA_HOME = $CatalinaHomeLocation
```

Comment:
Actually removing this line was a BAD idea since [System.Environment]::SetEnvironmentVariable(..."Machine") doesn't actually set the local process environment. :( I have a change I'm testing now.